create_window_sys/src/main.rs

```diff
@@ -1,66 +1,66 @@
 use windows_sys::{
-    Win32::Foundation::*, Win32::Graphics::Gdi::ValidateRect, Win32::Graphics::Gdi::*,
-    Win32::System::LibraryLoader::GetModuleHandleA, Win32::UI::WindowsAndMessaging::*,
+    Win32::Graphics::Gdi::ValidateRect, Win32::System::LibraryLoader::GetModuleHandleA,
+    Win32::UI::WindowsAndMessaging::*,
 };
 
 fn main() {
     unsafe {
-        let instance = GetModuleHandleA(PSTR(std::ptr::null_mut()));
-        debug_assert!(instance.0 != 0);
+        let instance = GetModuleHandleA(std::ptr::null_mut());
+        debug_assert!(instance != 0);
 
-        let window_class = PSTR(b"window\0".as_ptr() as _);
+        let window_class = b"window\0".as_ptr() as _;
 
         let wc = WNDCLASSA {
-            hCursor: LoadCursorW(HINSTANCE(0), IDC_ARROW),
+            hCursor: LoadCursorW(0, IDC_ARROW),
             hInstance: instance,
             lpszClassName: window_class,
-            style: WNDCLASS_STYLES(CS_HREDRAW.0 | CS_VREDRAW.0),
-            lpfnWndProc: wndproc,
+            style: CS_HREDRAW | CS_VREDRAW,
+            lpfnWndProc: Some(wndproc),
             cbClsExtra: 0,
             cbWndExtra: 0,
-            hIcon: HICON(0),
-            hbrBackground: HBRUSH(0),
-            lpszMenuName: PSTR(std::ptr::null_mut()),
+            hIcon: 0,
+            hbrBackground: 0,
+            lpszMenuName: std::ptr::null_mut(),
         };
 
         let atom = RegisterClassA(&wc);
         debug_assert!(atom != 0);
 
         CreateWindowExA(
-            WINDOW_EX_STYLE(0),
+            0,
             window_class,
-            PSTR(b"This is a sample window".as_ptr() as _),
-            WINDOW_STYLE(WS_OVERLAPPEDWINDOW.0 | WS_VISIBLE.0),
+            b"This is a sample window".as_ptr() as _,
+            WS_OVERLAPPEDWINDOW | WS_VISIBLE,
             CW_USEDEFAULT,
             CW_USEDEFAULT,
             CW_USEDEFAULT,
             CW_USEDEFAULT,
-            HWND(0),
-            HMENU(0),
+            0,
+            0,
             instance,
             std::ptr::null_mut(),
         );
 
         let mut message = std::mem::zeroed();
 
-        while GetMessageA(&mut message, HWND(0), 0, 0).0 != 0 {
+        while GetMessageA(&mut message, 0, 0, 0) != 0 {
             DispatchMessageA(&mut message);
         }
     }
 }
 
-extern "system" fn wndproc(window: HWND, message: u32, wparam: WPARAM, lparam: LPARAM) -> LRESULT {
+extern "system" fn wndproc(window: isize, message: u32, wparam: usize, lparam: isize) -> isize {
```

Comment:
Isn't there still a type alias for HWND in windows-sys?

Reply:
Yes, I suppose the samples would be clearer if they were using those. 👍🏼